### DISCUSSION TITLE
On the search search templates should we indicate whether there are active filters

### DISCUSSION THREAD

**Discussion by jaymcp** [amnestywebsite/humanity-theme#73] [Enhancement Requests]
Because the taxonomy term filters are hidden by default in the new search UI, it may be beneficial to indicate to the user that they have active filters in the hidden area.

We could do this by adding a number into the button that expands to reveal the filters:
![](http://bigbite.im/i/bs7joC+)

Further, we could indicate within which taxonomies the user has filters active, by updating the taxonomy label(s):
![](http://bigbite.im/i/nt9R5U+)

**Comment by AmnestyAM** [comment]
nice idea @jonmcp I saw this too and though if used we could keep open, will keep this for a future enhancement! Thanks
